src/main/resources/log4j2.xml

```diff
@@ -1,11 +1,41 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Configuration status="WARN">
+  <Properties>
+    <Property name="csvLog.fileName">metric</Property>
+    <Property name="file-header"></Property>
+  </Properties>
   <Appenders>
+    <RollingFile name="csvFileServer"
```

Comment:
minor, but call it "metricFileServer"
idea is to describe what is does but not how it's done

same for the client rolling file